NOTICE

```diff
@@ -10,6 +10,9 @@ within this component.
 Copyright 2014 Cisco Systems, Inc.
 Licensed under the Apache License, Version 2.0
 
+ISC License
```

Comment:
Sorry, crossed line here: please delete lines 13-15 and put them instead into the file called LICENSE before the text of the ISC license.  The LICENSE file is in the same directory as NOTICE.